package.json

```diff
@@ -62,9 +62,13 @@
     "@adonisjs/shield": "1.1.0",
     "@adonisjs/validator": "5.1.0",
     "@electron/remote": "2.0.10",
+    "@emotion/react": "11.11.1",
+    "@emotion/styled": "11.11.0",
     "@krisdages/electron-process-manager": "3.0.0",
     "@mdi/js": "7.2.96",
     "@mdi/react": "1.6.1",
+    "@mui/icons-material": "5.14.7",
+    "@mui/material": "5.14.7",
```

Comment:
You might not be aware of it, but you are pulling in complete new UI and icon libraries here. Usually on a frontend project, you try to be consistent and not mix and match the look of various UI elements and icons.

Apart from that, https://bundlephobia.com/package/@mui/material@5.14.7 is really heavy in terms of bundle size.

Reply:
I noticed that while coding but for some reason I don't recall (I was night owl coding XD) I made the decision to import that .. probably because I already added the mui library before. 

But I agree that if we can, we should refactor the code... mainly because of bundle size, as you referred